libs/fluid-elements/icon/README.md

```diff
@@ -0,0 +1,8 @@
+# fluid-icon
+
+This library was generated with [Nx](https://nx.dev).
+
+## Running unit tests
+
+Run `ng test fluid-icon` to execute the unit tests via
```

Comment:
```suggestion
Run `bazel test //libs/fluid-elements/icon:test` to execute the unit tests via
```

Reply:
@heartdisease why have you resolved it? it is not incorporated yet.

Reply:
Seems like Github messed up the "Commit suggestion" operation...